charts/topolvm/values.yaml

```
@@ -150,6 +150,11 @@ lvmd:
   # lvmd.args -- Arguments to be passed to the command.
   args: []
 
+  # lvmd.securityContext. -- Container securityContext.
```

Could you delete this period to generate README.md correctly?

```suggestion
  # lvmd.securityContext -- Container securityContext.
```

---

Done! I was trying to stick to what already exists from `node.securityContext`.